lib/components/app/responsive-webapp.js

```diff
@@ -218,14 +217,10 @@ class RouterWrapper extends Component {
               }}
             />
             <Route
-              // This route is called immediately after login.
+              // This route is called immediately after login
+              // and also by the auth0 redirect callback.
+              // The component for this route determines what is ultimately shown after login.
```

Comment:
This comment feels too cryptic. Instead of `The component for this route determines what is ultimately shown after login.`, just describe the states that AfterSignInSwitch will present.

Reply:
Updated